### DISCUSSION TITLE
Octodash on Surface Pro 3 running Ubuntu

### DISCUSSION THREAD

**Discussion by maxime-granger** [UnchartedBull/OctoDash#1559] [Q&A] [answered]
Just Curious : 
Is it doable to run Octodash on Ubuntu installed on a Surface Pro 3 ? 

Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   36 bits physical, 48 bits virtual
CPU(s):                          4
On-line CPU(s) list:             0-3
Thread(s) per core:              1
Core(s) per socket:              4
Socket(s):                       1
NUMA node(s):                    1
Vendor ID:                       GenuineIntel
CPU family:                      6
Model:                           76
Model name:                      Intel(R) Atom(TM) x7-Z8700  CPU @ 1.60GHz
Stepping:                        3
CPU MHz:                         629.276
CPU max MHz:                     2400.0000
CPU min MHz:                     480.0000
BogoMIPS:                        3200.00
Virtualization:                  VT-x
L1d cache:                       96 KiB
L1i cache:                       128 KiB
L2 cache:                        2 MiB
NUMA node0 CPU(s):               0-3

**Comment by UnchartedBull** [comment, marked as answer]
I don't see any reason why it shouldn't run. But maybe follow the manual instructions and skip all the ratpoison part since you most likely already have a Desktop Manager running.
